packages/angular/tsconfig.schematics.json

```diff
@@ -12,7 +12,8 @@
     "pretty": true,
     "rootDir": "src/schematics",
     "target": "es2017",
-    "outDir": "dist/schematics"
+    "outDir": "dist/schematics",
+    "skipLibCheck": true
```

Comment:
This isn't the correct solution.

I'm getting the following error on build. I suspect it has something to do with mismatched/multiple versions of node or typescript, but I'm not 100% sure yet. I was able to resolve a few other errors by updating to `@types/node@14`. 

```
> @ionic/angular@7.8.0 build.core /Users/liamdebeasi/Ionic/ionic/packages/angular
> node scripts/build-core.js

../../../node_modules/.pnpm/@angular-devkit+schematics@17.2.3/node_modules/@angular-devkit/schematics/src/engine/engine.d.ts:9:23 - error TS2688: Cannot find type definition file for '@types/node/ts4.8/url'.

9 /// <reference types="@types/node/ts4.8/url" />
                        ~~~~~~~~~~~~~~~~~~~~~

../../../node_modules/.pnpm/@angular-devkit+schematics@17.2.3/node_modules/@angular-devkit/schematics/src/engine/interface.d.ts:9:23 - error TS2688: Cannot find type definition file for '@types/node/ts4.8/url'.

9 /// <reference types="@types/node/ts4.8/url" />
                        ~~~~~~~~~~~~~~~~~~~~~
```

Reply:
Actually, according to https://angular.io/guide/schematics-for-libraries#building-your-schematics this is the recommended solution.